UPGRADE-1.10.md

```diff
@@ -1,3 +1,10 @@
+# UPGRADE FROM `v1.10.8` TO `v1.10.10`
+
+1. Field `byGuest` has been added to `Sylius\Component\Core\Model\Order`, this change will allow us to distinguish carts 
+between guests and logged in customers.
+
+2. Not passing `tokenStorage` through constructor is deprecated in Sylius 1.10.9 and it will be removed in Sylius 2.0.
```

Comment:
What constructor? 😄 

```suggestion
2. Not passing `tokenStorage` through `Sylius\Component\Core\Cart\Context\ShopBasedCartContext` constructor is deprecated in Sylius 1.10.9 and it will be removed in Sylius 2.0.
```